nifi-nar-bundles/nifi-extension-utils/nifi-conflict-resolution/src/main/java/org/apache/nifi/processors/conflictresolution/ConflictResolutionStrategy.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.conflictresolution;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.nifi.components.DescribedValue;
+
+public enum ConflictResolutionStrategy implements DescribedValue {
+    FAIL("fail", "fail", "Handle file conflict as failure."),
```

Comment:
I agree that we need to keep the lowercase values for compatibility with current usage, however, this could be simplified to two arguments instead of three, and `getDisplayName()` can return `value`.
```suggestion
    FAIL("fail", "Handle file conflict as failure."),
```